aea/protocols/dialogue/base.py

```diff
@@ -353,6 +352,7 @@ def __init__(
         self._message_class = message_class
         self._terminal_state_callbacks: Set[Callable[["Dialogue"], None]] = set()
         self._last_message_id: Optional[int] = None
+        self._ordered_message_ids: List[int] = []
```

Comment:
This is useful to keep track of, otherwise we cannot debug the order in which messages where observed by this dialogue.

Reply:
probably can reconstruct it from messages stored?
cause  have ids and targets

Reply:
But they don't tell us the full order.